external/fmdb

```diff
@@ -1 +1 @@
-Subproject commit 243919a3529fa8de95052b2fc2994bfd931a618c
+Subproject commit 81d04e793839d3f3f7756004cd0f7b1d6db394ee
```

Comment:
Had to expose some headers as public and tweak the public headers folder path in the project file - hence the changes in ```FMDB``` fork.